tests/e2e/specs/drawer.cy.js

```diff
@@ -35,4 +38,13 @@ describe('Drawer component', () => {
       .get('#toggle-drawer')
       .should('be.visible')
   })
+  it('should drag to trigger resize', () => {
+    cy.get('.v-navigation-drawer').invoke('innerWidth').should('be.lt', 360)
+    cy.get('.v-navigation-drawer__border')
+      .trigger('mousedown', { which: 1 })
+      .trigger('mousemove', { clientX: 100 })
+      .trigger('mouseup', { force: true })
+    cy.get('.v-navigation-drawer__border')
+      .invoke('innerWidth').should('be.lt', 460)
+  })
```

Comment:
This test does not seem to be quite working unfortunately. It does not appear to resize 

Before mousemove:

![image](https://user-images.githubusercontent.com/61982285/189864589-196d5def-1452-4514-abed-aac7eaa10ee1.png)

After mousemove:

![image](https://user-images.githubusercontent.com/61982285/189864826-339b4a4a-1566-4acf-8435-fc5e85e396ec.png)

Reply:
Theres something really weird going on here, im looking into it.

Reply:
Fixed with latest test updates ( think )